api/src/domain-logic/analytics.js

```diff
@@ -0,0 +1,61 @@
+const dayjs = require('dayjs')
+
+const supabase = require('./supabase')
+
+const today = () => dayjs().format('YYYY-MM-DD')
+const TABLE =
+  process.env.NODE_ENV === 'production' ? 'analytics' : 'analytics_dev'
+
+/* Note
+ * - Cannot use upsert because in order to make upsert correct, should use id instead
+ */
+
+const updateAnalytic = ({ count, ...item }) =>
+  supabase
+    .from(TABLE)
+    .update(
+      { ...item, count: count + 1 },
```

Comment:
We should always be specific about which columns to get update, rather than spreading an object into it. Who knows which properties that object contains?